src/content/docs/en/recipes/i18n.mdx

```diff
@@ -0,0 +1,322 @@
+---
+title: Add internationalization features 
+description: Use dynamic routing and content collections to add i18n support to your Astro site.
+type: recipe
+i18nReady: false
+---
+import FileTree from '~/components/FileTree.astro'
+import StaticSsrTabs from '~/components/tabs/StaticSsrTabs.astro'
+
+Astro doesn't have built-in internationalization (i18n) support, but you can build your own i18n solution. In this recipe, you'll learn how to use content collections and dynamic routing to serve content for different languages.
+
+## Recipe
+
+### Set up pages for each language
+
+1. Create a directory for each language you want to support. For example, `en/` and `fr/` if you are supporting English and French:
+
+    <FileTree>
+    - src/
+      - pages/
+        - **en/**
+          - about.astro
+          - index.astro
+        - **fr/**
+          - about.astro
+          - index.astro
+        - index.astro
+    </FileTree>
+
+    
+2. Set up `src/pages/index.astro` to redirect to your default language.
+
+    <StaticSsrTabs>
+      <Fragment slot="static">
+        ```astro
+        ---
+        // src/pages/index.astro
+        ---
+        <meta http-equiv="refresh" content="0;url=/en/" />
+        ```
+
+        This approach uses a [meta refresh](https://en.wikipedia.org/wiki/Meta_refresh) and will work however you deploy your site. Some static hosts also let you configure server redirects with a custom configuration file. See your deploy platform’s documentation for more details.
+      </Fragment>
+      
+      <Fragment slot ="ssr">
+        If you are using an SSR adapter, you can use [`Astro.redirect`](/en/guides/server-side-rendering/#astroredirect) to redirect to the default language on the server.
+
+        ```astro
+        ---
+        // src/pages/index.astro
+        return Astro.redirect('/en/');
+        ---
+        ```
+      </Fragment>
+    </StaticSsrTabs>
+
+### Use collections for translated content
+
+1. Create a folder in `src/content/` for each type of content you want to include, and create subdirectories for each supported language. For example, to support English and French blog posts:
```

Comment:
The comma here is unnecessary, though removing it will expose the double "create". Considering this, I believe that just changing the second one to "add" solves the problem.
```suggestion
1. Create a folder in `src/content/` for each type of content you want to include and add subdirectories for each supported language. For example, to support English and French blog posts:
```